### PR TITLE
test for comment title suggestion, should fail

### PR DESCRIPTION
**Pull request by jywarren**

### CODE REVIEW

**Comment by gauravano** [comment]
Hey @jywarren, please update the status of this issue and can you please explain what's the goal here. Thanks!

**Comment by SidharthBansal** [comment]
@jywarren we can include this in GCI if you want
cc\ @Uzay-G

**Comment by SidharthBansal** [comment]
@VladimirMikulic

**Comment by VladimirMikulic** [comment]
Why not :)

**Comment by SidharthBansal** [comment]
@VladimirMikulic don't get overwhelmed. Please finish your previous work. Do things one by one else you will be perplexed by X branches at your terminal all for tests. lol...

**Comment by VladimirMikulic** [comment]
@SidharthBansal No, I won't, I am organized.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -102,7 +102,7 @@ def render_map(lat, lon)
   def title_suggestion(comment)
     comment.body.gsub(/\[propose:title\](.*?)\[\/propose\]/) do
       a = ActionController::Base.new
-      is_creator = current_user.drupal_user == Node.find(comment.nid).author
+      is_creator = @current_user.drupal_user == Node.find(comment.nid).author
```
Comment (Uzay-G) [inline app/helpers/application_helper.rb]: I think we need to remove the `@` here no? Some of the tests are failing because `current_user` is undefined.

**Comment by jywarren** [comment]
OK, abandoning this, but noting errors were:

```
ERROR["test_render_propose_title_template_when_author_is_logged_in", #<Minitest::Reporters::Suite:0x00000000122dbb50 @name="NotesControllerTest">, 45.679375883000006]
 test_render_propose_title_template_when_author_is_logged_in#NotesControllerTest (45.68s)
Minitest::UnexpectedError:         ArgumentError: wrong number of arguments (given 3, expected 1)
            test/functional/notes_controller_test.rb:1078:in `block in <class:NotesControllerTest>'
 FAIL["test_render_propose_title_template_when_author_is_not_logged_in", #<Minitest::Reporters::Suite:0x00000000125be778 @name="NotesControllerTest">, 45.69181685800001]
 test_render_propose_title_template_when_author_is_not_logged_in#NotesControllerTest (45.69s)
        Expected response to be a <2XX: success>, but was a <404: Not Found>
        test/functional/notes_controller_test.rb:1094:in `block in <class:NotesControllerTest>'
```

**Pull request closed without merging**